lib/daru/core/group_by.rb

```diff
@@ -58,6 +58,20 @@ def last
         tail(1)
       end
 
+      def summarize(options={})
+        colmn_value = []
+        options.keys.each do |vec|
+          do_this_on_vec = options[vec]
+          colmn_value << if @non_group_vectors.include?(vec)
```

Comment:
If `vec` is a vector from the `dataframe`, then apply the method `do_this_on_vec ` on the `vec` column using `apply_method_on_colmns `. 
Else it is required to use the `dataframe` to apply the method (for this `apply_method_on_df ` is used).